headeronly_src/sqlite3pp.h

```diff
@@ -294,9 +294,12 @@ namespace sqlite3pp
     };
 
     class query_iterator
-      : public std::iterator<std::input_iterator_tag, rows>
     {
      public:
+      using iterator_category = std::input_iterator_tag;
+      using value_type = rows;
+      using difference_type = void;
```

Comment:
And increase the version number to 1.0.9

Reply:
ok!